### PR TITLE
docs(trace): traceability kit release for 23.09 (3.2.0)

### PR DESCRIPTION
**Pull request by mhellmeier**
This pull request updates the Traceability KIT to be compatible with the Catena-X release 3.2.0.

### CODE REVIEW

File: docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 1]
```diff
@@ -386,55 +442,284 @@ If no access control shall be applied, externalSubjectId must be omitted (no acc
...
+        "securityAttributes": [ 
+          { "type": "NONE", "key": "NONE", "value": "NONE" }
+        ]
+    }
+}
+```
+
+The following conventions apply for the endpoint:
+
+- `interface`, `endpointProtocol`, `endpointProtocolVersion`, `subprotocol`, `subprotocolBodyEncoding`, and `securityAttributes` are set as defined in the CX-0002 standard.
```
Comment (SebastianBezold) [inline docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md:598]: Is the `CX-002` Standard already publicly available? If so, I think a link would help the reader. 
Potentially all references to the CX standards should link to the source.

File: docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md [hunk 2]
```diff
@@ -386,55 +442,284 @@ If no access control shall be applied, externalSubjectId must be omitted (no acc
...
+- `subprotocolBody`: a semicolon-separated list of parameters used to negotiate the required contract agreement.
+  - `id=123`: The ID of the EDC asset for which a contract negitiation should be intiated. This ID is also called dataset ID as it is stored as `https://www.w3.org/ns/dcat/dataset.@id` in a catalog entry. This ID must be set by the data provider when creating the asset. Do not confuse this EDC asset ID (dataset ID) with other IDs that might be defined additionally for an EDC asset, e.g., `https://w3id.org/edc/v0.0.1/ns/id` (often refered to as `edc:id`).
+  - `dspEndpoint`: Server and port of the EDC control plane used for contract negotiation.
+
+> :raised_hand: **Backend Data Service for Submodels**
+According to CX-0002, the backend data service identified via `href`and the filter criteria in `subprotocolBody` MUST be conformant to the Asset Administration Shell Profile SSP-003 of the Submodel Service Specification and must at least support the logical operation GetSubmodel. In release 3.2, only the logical parameter Content=Value must be supported via path suffix "/submodel/$value". This might change in later Catena-X releases.
```
Comment (SebastianBezold) [inline docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md:608]: I think here the Tractus-X release is meant, right?
As I understood it, the Standards are published at the Catena-X assosiation, but the models are hosted in Eclipse Tractus-X. 
Could be a good thing to clarify and also share with other teams

File: docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md (hunk 1 above)
Comment (mhellmeier) [reply docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md]: I just added links to the location of the standards.

File: docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md (hunk 2 above)
Comment (mhellmeier) [reply docs-kits/kits/Traceability Kit/Software Development View/page_software-development-view.md]: We are talking about the Catena-X release. I updated the section to clarify it.

**Comment by SebastianBezold** [comment]
From my point of view just a view minor things, that would not block a merge, but could be considered for further improvement:

- Clarify references to "Catena-X release". In my point of view, there is no Catena-X release and "Tractus-X release" is meant
- Add a few more reference links. I.e. Standards in it's version on catena-x.net (changelog + in text)

**Review by SebastianBezold — APPROVED** [review]
LGTM overall, but cannot provide feedback on the contents, so further approvals should be given

**Comment by mhellmeier** [comment]
Based on @SebastianBezold comments:
> * Clarify references to "Catena-X release". In my point of view, there is no Catena-X release and "Tractus-X release" is meant

I updated the files accordingly to only mention "release" instead of "Catena-X release" to reduce misunderstandings.


> * Add a few more reference links. I.e. Standards in it's version on catena-x.net (changelog + in text)

I updated and added some links to the standards. Nevertheless due to the current timeline, it isn't possible to link to the correct standard. As I understand, @danielmiehle will discuss this in the lifecycle process.

**Comment by mhellmeier** [comment]
This pull request is still in Draft mode since it waits for internal approval of the traceability use case team as well as a successful quality gate check.

If both things are fulfilled, I will change it to be ready for the merge.

**Review by maximilianong — COMMENTED** [review]
Can you please add the versions of the standards you are referencing to
(e.g. CX - 0004 Semantic Model V 1.0.1)

**Review by maximilianong — APPROVED** [review]
After IP clearance - PR will be merged.

**Comment by mhellmeier** [comment]
> After IP clearance - PR will be merged.

Just to let you know, IP clearance has been successful. The branch is updated to the current status of `main` accordingly. All kit changes are integrated into the new versioned structure. Ready to be merged, @maximilianong!

**Pull request merged**